packages/xenstore_transport/xenstore_transport.1.3.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "dave@recoil.org"
+authors: [
+  "Christian Lindig"
+  "David Scott"
+  "Euan Harris"
+  "John Else"
+  "Jon Ludlam"
+  "Jonathan Davies"
+  "Marcello Seri"
+  "Si Beaumont"
+  "Thomas Sanders"
+  "Vincent Bernardoff"
+]
+license: "LGPL-2.1-only"
+tags: ["org:mirage" "org:xapi-project"]
+homepage: "http://github.com/xapi-project/ocaml-xenstore-clients"
+doc: "http://xapi-project.github.io/ocaml-xenstore-clients"
+bug-reports: "http://github.com/xapi-project/ocaml-xenstore-clients/issues"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "ocamlfind" {build}
```

Comment:
Do you really depend on ocamlfind? I could not find any use of it in your repository but maybe I missed something

Reply:
I don't think the package depends on ocamlbuild